src/anybuf.rs

```diff
@@ -399,19 +420,41 @@ impl Anybuf {
     ///
     /// Use this instead of multiple [`Anybuf::append_bytes`] to ensure empty values are not lost.
     ///
+    /// The generic type `B` is the type of a single element in the input slice. This
+    /// is typically something like `&[u8]`, `Vec<u8>` but also `&str` and `String` work.
+    ///
     /// ## Example
     ///
     /// ```
     /// # use anybuf::Anybuf;
     /// let blob = vec![4u8; 75];
     ///
-    /// // Three bytes fields with field number 5
+    /// // Three bytes fields with field number 5, slice of slices
+    /// let serialized = Anybuf::new()
+    ///     .append_repeated_bytes(5, &[blob.as_slice(), b"", b"abcd"])
+    ///     .into_vec();
+    ///
+    /// // Works with Vec<u8> elements too
+    /// let owned: Vec<Vec<u8>> = vec![
+    ///     vec![1u8; 10],
+    ///     vec![2u8; 10],
+    ///     vec![3u8; 10],
+    /// ];
```

Comment:
The nice thing about this example is that it also works the same way for `Vec<cosmwasm_std::Binary>` and `Vec<cosmwasm_std::HexBinary>`